governance/default.go

```diff
@@ -927,7 +938,5 @@ func AddGovernanceCacheForTest(g *Governance, num uint64, config *params.ChainCo
 	// Don't update cache if num (block number) is smaller than the biggest number of cached block number
 
 	data := getGovernanceItemsFromChainConfig(config)
-	cKey := getGovernanceCacheKey(num)
-	g.itemCache.Add(cKey, data.Items())
```

Comment:
In the last commit, 930 and 931 has been removed, is it intended?

Reply:
@ehnuje Yes, it was intended. Because now the function calls g.addGovernanceCache(), those two removed lines is being done in the addGovernanceCache().